src/libcore/tests/hash/sip.rs

```diff
@@ -299,14 +299,17 @@ fn test_hash_no_concat_alias() {
     let u = ("a", "abb");
 
     assert!(s != t && t != u);
```

Comment:
this looks like it should have been removed as well?

Reply:
Correct, good catch! I've removed that assert.